eclair-core/src/main/scala/fr/acinq/eclair/db/sqlite/SqliteNetworkDb.scala

```diff
@@ -171,8 +170,7 @@ class SqliteNetworkDb(sqlite: Connection) extends NetworkDb with Logging {
   override def isPruned(shortChannelId: ShortChannelId): Boolean = withMetrics("network/is-pruned", DbBackends.Sqlite) {
     using(sqlite.prepareStatement("SELECT short_channel_id from pruned WHERE short_channel_id=?")) { statement =>
       statement.setLong(1, shortChannelId.toLong)
-      val rs = statement.executeQuery()
-      rs.next()
+      statement.executeQuery().next()
```

Comment:
Same here, is this correctly calling `hasNext` before calling `next()` internally?

Reply:
Fixed with 305d0914d.